theme/less/main.less

```diff
@@ -23,6 +23,26 @@ main
   margin-right: auto;
 }
 
+.greyscale
+{
+  filter: grayscale(100%);
```

Comment:
Detail: Lidehacker logo looks more lighter than others, feels bit off?

Reply:
Yes, bit I think it's from colors of the current logo of Lifehacker. Because in case of c-net we don't observe such light greyscale.

Reply:
You think there are any other filters that can work better for our case? Make sense to do bit of research on that regard after converting all logos to using filter.

Reply:
Suggestion: We can use 
```
#media a {
filter: grayscale(100%);
opacity: 0.7;
}
```
instead of classes on anchors.

Reply:
> 
> 
> You think there are any other filters that can work better for our case? Make sense to do bit of research on that regard after converting all logos to using filter.

We can  apply `filter: grayscale(100%) brightness(0.4);` filters on light-colored logos for example.

Reply:
I was thinking about a solution that doesn't requires exception for items, also it's hard to programmatically calculate light-colored logos.

Reply:
If you can't find a better filter you can apply on al items, then keep as it is.